core/client/src/lib.rs

```diff
@@ -91,9 +89,9 @@ pub mod light;
 #[cfg(feature = "std")]
 mod leaves;
 #[cfg(feature = "std")]
-mod call_executor;
+pub mod call_executor;
```

Comment:
```suggestion 
mod call_executor;
```
The required trait etc is already exported some lines below.